packages/suite/src/components/suite/modals/ReduxModal/TransactionReviewModal/TransactionReviewOutputList/TransactionReviewOutputList.tsx

```
@@ -2,287 +2,144 @@ import { useEffect, useRef } from 'react';
-                            <TransactionReviewTotalOutput
-                                ref={totalRef}
+        <Column gap={spacings.md}>
+            {outputs.map((output, index) => {
+                const isHeadingShown =
+                    isMultirecipient && (output.type === 'address' || index === summaryIndex);
+                const recipientIndex = outputs
+                    .filter(({ type }) => type === 'address')
+                    .indexOf(output);
+
+                return (
+                    <Wrapper key={index} ref={ref => (outputRefs.current[index] = ref)}>
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)